radio/src/targets/simu/opentxsimulator.cpp

```diff
@@ -95,7 +95,7 @@ void OpenTxSimulator::start(QByteArray & ee, bool tests)
 
 void OpenTxSimulator::start(const char * filename, bool tests)
 {
-#if defined(PCBSKY9X) && !defined(REVX)
+#if defined(PCBSKY9X) && !defined(REVX) && !defined(AR9X)
```

Comment:
I just checked, you should use NUM_ROTARY_ENCODERS > 0 and something is missing here:
#if defined(PCBSKY9X) && (defined(REVX) || defined(AR9X)
  #define MAX_ROTARY_ENCODERS          1
  #define NUM_ROTARY_ENCODERS          0
#elif defined(PCBSKY9X)
  #define MAX_ROTARY_ENCODERS          1
  #define NUM_ROTARY_ENCODERS          1
...